src/routes/Book/Books.ts

```diff
@@ -0,0 +1,83 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response, Router } from 'express';
+
+import { paramMissingError, IRequest } from '@shared/constants';
+
+import bookModel from '../../entities/Book/Book.schema';
+
+const router = Router();
+const { BAD_REQUEST, CREATED, OK } = StatusCodes;
+
+const Book = bookModel;
+
+
+
+/******************************************************************************
+ *                      Get All Users - "GET /api/books/all"
+ *****************************************************************************/
+
+router.get('/all', async (req: Request, res: Response) => {
```

Comment:
Could we change this endpoint to /api/books (instead of /api/books/all) ?

As I understand it could be done under MNG-137 task. Am I right? :)

Reply:
yes, yes do as you think is right.
I commented all.
It was there only to check if schemas works ;)